internal/jobservice/eventstojobs/manage_subs_test.go

```diff
@@ -36,6 +36,9 @@ func (m *MockEventClient) Recv() (*api.EventStreamMessage, error) {
 	m.messagesSent += 1
 
 	if m.messagesSent > 3 {
+		// Sleep a bit to mimick the stream being open but no events
+		// currently available.
+		time.Sleep(time.Second * 5)
```

Comment:
Maybe we should put this under a little load and verify that this won't be flaky?

Reply:
Can we cross that bridge if it happens? Right now they don't pass at all.